superset-frontend/src/dashboard/components/FilterIndicatorsContainer.jsx

```diff
@@ -101,6 +101,15 @@ export default class FilterIndicatorsContainer extends React.PureComponent {
                 chartId,
                 column: name,
               });
+
+              // filter values could be single value or array of values
+              const values =
+                isNil(columns[name]) ||
+                (isDateFilter && columns[name] === 'No filter') ||
+                (Array.isArray(columns[name]) && columns[name].length === 0)
```

Comment:
isEmpty(number) will return true, this caused indicator didn't show the selected filter value.